orocos_kdl/src/chainiksolvervel_pinv_nso.cpp

```diff
@@ -121,18 +123,20 @@ namespace KDL
             A += qd*qd * weights(i)*weights(i);
         }
 
-        // Calculate inverse Jacobian Jc^-1
-        for (i = 0; i < nj; ++i) {
-            tmp(i) = weights(i)*(q_in(i) - opt_pos(i)) / A;
-        }
+        if (A > 1e-9) {
```

Comment:
I know we have more magic numbers like this in KDL, but could you elaborate on this one? Why 1e-9 and not 1e-12 like in other places?